Sources/AppStoreConnectCLI/Services/Operations/ExpireBuildOperation.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ExpireBuildOperation: APIOperation {
+
+  struct Options {
+    let buildId: String?
```

Comment:
Build id should be non optional ideally
I would expect`ReadBuildOperation` to return a single non optional model
It should take in one build number according to #113